src/Microsoft.OData.Core/Json/IStreamBasedJsonWriterFactory.cs

```diff
@@ -11,7 +11,7 @@
 namespace Microsoft.OData.Json
 {
     /// <summary>
-    /// The interface of factories that create <see cref="IJsonWriter"/> instances
+    /// The interface of factories that create <see cref="IJsonWriter"/> and <see cref="IJsonWriterAsync"/> instances

```

Comment:
There's a full stop, the statement ends on the following line. Here's the full summary:

```c#
/// <summary>
/// The interface of factories that create <see cref="IJsonWriter"/> and <see cref="IJsonWriterAsync"/> instances
/// that write directly to a <see cref="Stream"/> rather than a <see cref="TextWriter"/>.
/// </summary>
```